spec/graphql/__snapshots__/range_filter_search_query.snap

```diff
@@ -1 +1 @@
study_type":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"study_type":{"terms":{"field":"study_type","size":10}}}},"sponsors":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"sponsors":{"terms":{"field":"sponsors","size":10}}}},"browse_condition_mesh_terms":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"browse_condition_mesh_terms":{"terms":{"field":"browse_condition_mesh_terms","size":10}}}},"phase":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"phase":{"terms":{"field":"phase","size":10}}}},"rating_dimensions":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"rating_dimensions":{"terms":{"field":"rating_dimensions","size":10}}}},"browse_interventions_mesh_terms":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"browse_interventions_mesh_terms":{"terms":{"field":"browse_interventions_mesh_terms","size":10}}}},"interventions_mesh_terms":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"interventions_mesh_terms":{"terms":{"field":"interventions_mesh_terms","size":10}}}},"front_matter_keys":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"front_matter_keys":{"terms":{"field":"front_matter_keys","size":10}}}},"start_date":{"filter":{"bool":{"must":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"aggs":{"start_date":{"terms":{"field":"start_date","size":10}}}}},"query":{"bool":{"must":{"query_string":{"query":"stomach"}},"filter":[{"bool":{"must":[{"bool":{"filter":[{"bool":{"should":[]}}]}},{"bool":{"filter":[{"range":{"start_date":{"from":"2010-01-01T05:00:00.000Z","include_lower":true}}}]}}]}}]}},"sort":[{"nct_id":"asc"}],"timeout":"11s","_source":false,"size":25,"from":25}
```

Comment:
these changed because i learned we were doing a search ahead of time to get front matter keys before performing a search.